Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/ColorPicker.cpp

```diff
@@ -708,6 +726,40 @@ void ColorPicker::setComment(QString comment)
     m_commentLabel->setText(comment);
 }
 
+void ColorPicker::setAlternateColorspaceEnabled(bool enabled)
+{
+    if (enabled && m_alternateColorSpaceInfoLayout->isEmpty())
+    {
+        m_alternateColorSpaceInfoLayout->addWidget(m_alternateColorSpaceIntLabel, 0, 0);
+        m_alternateColorSpaceInfoLayout->addWidget(m_alternateColorSpaceFloatLabel, 1, 0);
+        m_alternateColorSpaceInfoLayout->addWidget(m_alternateColorSpaceIntValue, 0, 1);
+        m_alternateColorSpaceInfoLayout->addWidget(m_alternateColorSpaceFloatValue, 1, 1);
+    }
+
+    if (!enabled && !m_alternateColorSpaceInfoLayout->isEmpty())
+    {
+        m_alternateColorSpaceInfoLayout->removeWidget(m_alternateColorSpaceIntLabel);
+        m_alternateColorSpaceInfoLayout->removeWidget(m_alternateColorSpaceFloatLabel);
+        m_alternateColorSpaceInfoLayout->removeWidget(m_alternateColorSpaceIntValue);
+        m_alternateColorSpaceInfoLayout->removeWidget(m_alternateColorSpaceFloatValue);
+    }
+}
```

Comment:
It might be simpler to always leave them in the layout and use setVisible

Reply:
+1

Reply:
Hm, I didn't see a setVisible function.

Reply:
Ah setVisible is available on QWidget but not QLayout.